examples/sales_analytics/main.bal

```diff
@@ -0,0 +1,62 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/oauth2;
+import ballerinax/hubspot.crm.commerce.quotes as quotes;
+
+configurable string & readonly clientId = ?;
+configurable string & readonly clientSecret = ?;
+configurable string & readonly refreshToken = ?;
+
+public function main() returns error? {
+    
+    quotes:OAuth2RefreshTokenGrantConfig auth = {
+        clientId,
+        clientSecret,
+        refreshToken,
+        credentialBearer: oauth2:POST_BODY_BEARER
+    };
+
+    final string serviceUrl = "https://api.hubapi.com/crm/v3/objects/quotes";
+
+    final quotes:Client storeClient = check new quotes:Client(config = {auth}, serviceUrl = serviceUrl);
```

Comment:
As mentioned in the previous comment. Remove the default serviceUrl. 
Check and remove all the places where applicable.